scene/2d/audio_stream_player_2d.cpp

```diff
@@ -448,6 +471,29 @@ Ref<AudioStreamPlayback> AudioStreamPlayer2D::get_stream_playback() {
 	return stream_playback;
 }
 
+void AudioStreamPlayer2D::set_distance_drawing_enabled(bool enable) {
+	distance_drawing_enabled = enable;
+	notify_property_list_changed();
```

Comment:
Thus could be under TOOLS_ENABLED.

Reply:
A lot of stuff could be under `TOOLS_ENABLED` here, including the entirety of `_validate_property()` which isn't in basically any file that has it. There should a discussion sometime on how we should deal with this kind of stuff codebase-wise.

Reply:
I actually meant the `notify_property_list_changed()` call but ok.

Reply:
And I understood. I'm just mentioning the inconsistency of things that are and aren't encapsulated by `TOOLS_ENABLED`.